client/clip_client/client.py

```diff
@@ -184,10 +181,15 @@ def _iter_doc(self, content) -> Generator['Document', None, None]:
                 )
 
     def _get_post_payload(self, content, kwargs):
+        parameters = {}
+        if 'batch_size' in kwargs:
+            parameters['minibatch_size'] = kwargs['batch_size']
+
         return dict(
             on='/',
             inputs=self._iter_doc(content),
-            request_size=kwargs.get('batch_size', 8),
+            request_size=kwargs.get('batch_size', 32),
+            parameters=parameters,
```

Comment:
This seems a breaking change and undocumented? How does it affect here?

https://clip-as-service.jina.ai/user-guides/client/#control-batch-size

Reply:
Actually, the document is not correct. To control the batch size, the user needs to update `c.encode(..., batch_size=1024)`, and `minibatch_size` in `torch-flow.yml`. With this commit, the user doesn't need to care about `torch-flow.yml`